openedx/core/djangoapps/discussions/models.py

```diff
@@ -55,85 +55,158 @@ class Features(Enum):
+            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
+            Features.QUESTION_DISCUSSION_SUPPORT.value,
+            Features.COMMUNITY_TA_SUPPORT.value,
+            Features.EMAIL_NOTIFICATIONS.value,
+            Features.BLACKOUT_DISCUSSION_DATES.value,
+            Features.DISCUSSION_CONTENT_PROMPTS.value,
+            Features.DIRECT_MESSAGES_FROM_INSTRUCTORS.value,
+            Features.USER_MENTIONS.value,
+        ],
+        'documentation_urls': {
+            'learnMore': 'https://piazza.com/product/overview',
+            'configurationDocumentation': 'https://support.piazza.com/support/solutions/articles/48001065447-configure-piazza-within-edx',
+            'documentation': 'https://support.piazza.com/',
+            'accessibilityDocumentation': 'https://piazza.com/product/accessibility',
+            'emailId': 'team@piazza.com',
+        }
+    },
+    'edx-next': {
+        'features': [
+            Features.AUTOMATIC_LEARNER_ENROLLMENT.value,
+            Features.WCAG_2_0_SUPPORT.value,
+            Features.INTERNATIONALIZATION_SUPPORT.value,
+            Features.ANONYMOUS_POSTING.value,
+            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
+            Features.QUESTION_DISCUSSION_SUPPORT.value,
+            Features.COMMUNITY_TA_SUPPORT.value,
+            Features.EMAIL_NOTIFICATIONS.value,
+            Features.BLACKOUT_DISCUSSION_DATES.value,
+            Features.SIMPLIFIED_IN_CONTEXT_DISCUSSION.value,
+            Features.ADVANCED_IN_CONTEXT_DISCUSSION.value,
+            Features.COURSE_COHORT_SUPPORT.value,
+            Features.RESEARCH_DATA_EVENTS.value,
+            Features.DISCUSSION_CONTENT_PROMPTS.value,
+            Features.GRADED_DISCUSSIONS.value,
+        ],
+        'documentation_urls': {
+            'learnMore': '',
+            'configurationDocumentation': '',
+            'documentation': '',
+            'accessibilityDocumentation': '',
+            'emailId': '',
+        }
+    },
+    'yellowdig': {
+        'features': [
+            Features.WCAG_2_0_SUPPORT.value,
+            Features.ANONYMOUS_POSTING.value,
+            Features.REPORT_FLAG_CONTENT_TO_MODERATORS.value,
+            Features.QUESTION_DISCUSSION_SUPPORT.value,
+            Features.COMMUNITY_TA_SUPPORT.value,
+            Features.EMAIL_NOTIFICATIONS.value,
+            Features.RESEARCH_DATA_EVENTS.value,
+            Features.IN_PLATFORM_NOTIFICATIONS.value,
+            Features.GRADED_DISCUSSIONS.value,
+            Features.DIRECT_MESSAGES_FROM_INSTRUCTORS.value,
+            Features.USER_MENTIONS.value,
+        ],
+        'documentation_urls': {
+            'learnMore': 'https://www.youtube.com/watch?v=ZACief-qMwY',
+            'configurationDocumentation': '',
+            'documentation': 'https://hubs.ly/H0J5Bn70',
+            'accessibilityDocumentation': 'https://drive.google.com/file/d/1FT6E2ajMabFQI3NqgInPgGsQnLH7e2Mb/view?usp=sharing',
```

Comment:
Could be a security risk to have exposed links to google drive like this.

Reply:
@mehaknasir can we double-check with the reporter if this is the correct link? Very strange to have a Google Drive link for accessibility documentation.

Reply:
@awaisdar001 any thoughts on this.

Reply:
they'd need access - I don't think they are accessible by everyone with this URL?

Reply:
waiting on Marco to confirm this link.